gramps/plugins/lib/libgedcom.py

```diff
@@ -1562,10 +1558,15 @@ def __ansel_to_unicode(self, s):
         return ans
 
     def __init__(self, ifile, __add_msg):
-        BaseReader.__init__(self, ifile, "", __add_msg)
+        BaseReader.__init__(self, ifile, "ANSEL", __add_msg)
+        # In theory, we should have been able to skip the encode/decode from
```

Comment:
The comment doesn't stand alone; for ANSEL, we earlier converted from binary to text, and here are converting back to binary again so the remaining Ansel code below can convert back to text again.  Seems like we could have skipped the first encode/decode step , but would have lost the universal newline support (or had to code it ourselves).